src/pages/tutorials/v3.tsx

```diff
@@ -116,14 +114,14 @@ export default function tutorials({ data }: any) {
             link={`/tutorials/v3/frontier`}
           />
           <TutorialCard
-            title={`Cumulus Workshop`}
+            title={`Create a Parachain with Cumulus`}
             image={data.tutorialNine}
-            description={`A workshop to configure a Substrate node as a Parachain`}
+            description={`Learn to use the parachain template in local testnet`}
             time={`3 Hours`}
             difficulty={3}
             prerequisites={true}
-            version={`V3.0.0`}
-            link={`/tutorials/v3/cumulus-workshop/pt1`}
+            version={`polkadot-0.9.10`}
```

Comment:
using this version throughout for the workshop. it's a PITA to keep track of, as we change this on ~a weekly basis. Would be great if it was more confined to one location to specify.